R/calc_AUC.R

```diff
@@ -0,0 +1,99 @@
+require(caTools)
```

Comment:
what do you use caTools for? can you avoid using that somehow? if not, can you please use caTools::fun so it is more obvious where/what in caTools is used?